### ISSUE TITLE
Cannot import OPTForSequenceClassification on Kaggle notebooks (transformers 4.20.1, huggingface_hub 0.8.1)

### ISSUE THREAD

**Issue by navinelahi** [huggingface/transformers#18847]
### System Info

- `transformers` version: 4.20.1
- Platform: Linux-5.10.133+-x86_64-with-debian-bullseye-sid
- Python version: 3.7.12
- Huggingface_hub version: 0.8.1
- PyTorch version (GPU?): 1.11.0 (True)
- Tensorflow version (GPU?): 2.6.4 (True)
- Flax version (CPU?/GPU?/TPU?): 0.6.0 (gpu)
- Jax version: 0.3.16
- JaxLib version: 0.3.15
- Using GPU in script?: Yes
- Using distributed or parallel set-up in script?: No

Below is the code snippet and the error it caused. It can import OPTForCausalLM but not OPTForSequenceClassification
```
from transformers import OPTForCausalLM
from transformers import OPTForSequenceClassification, Trainer, TrainingArguments
```
**ImportError: cannot import name 'OPTForSequenceClassification' from 'transformers' (/opt/conda/lib/python3.7/site-packages/transformers/__init__.py)**

Any pointers to what might be wrong? Thank you so much. 

### Who can help?

@LysandreJik

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

from transformers import OPTForCausalLM
from transformers import OPTForSequenceClassification, Trainer, TrainingArguments

### Expected behavior

I would expect OPTForSequenceClassification to be imported normally.

**Comment by LysandreJik** [comment]
Hey @navinelahi, this class was released in v4.21; could you upgrade your `transformers` library`?

**Comment by navinelahi** [comment]
Yes I upgraded with `pip install transformers==4.21` and now it's working. Thank you so much. The issue is solved
